### PR TITLE
task1

### PR DESCRIPTION
**Pull request by DmitryChep**
Good evening my brother. Please check my task. I fixed my mistakes with spaces) and made the correct class hierarchy)

### CODE REVIEW

File: src/main/java/l2/Task1.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import l2.math.Gcd;
+
+public class Task1 {
+    public static void main (String[] args) {
+        Gcd mathGcd = new Gcd();
+
+        Scanner enterNum = new Scanner(System.in);
+        System.out.println("Hallo! Please enter in eny numbers :");
+        int number = enterNum.nextInt();
+        int a = number;
```
Comment (AlexChGit) [inline src/main/java/l2/Task1.java:13]: Please think how you can improve two lines of code above..

File: src/main/java/l2/Task1.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        System.out.println("Yes, and next number :");
+        int num = enterNum.nextInt();
+        int b = num;
```
Comment (AlexChGit) [inline src/main/java/l2/Task1.java:16]: similar to code above.. check the comment and take actions to improve it..

File: src/main/java/l2/Task1.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        System.out.println( "Result: "+ mathGcd.gcd(a, b ));
```
Comment (AlexChGit) [inline src/main/java/l2/Task1.java:17]: add a Space before Plus character

**Comment by AlexChGit** [comment]
Generally pretty clean code my brother, much better than previously. Continue in the same direction.

**Comment by DmitryChep** [comment]
Understood thanks. I'll do it now

**Pull request merged**